.github/workflows/ci.yml

```diff
@@ -42,8 +42,15 @@ jobs:
       - run: |
           julia --project=docs -e '
             using Pkg
+            Pkg.add("Documenter")
             Pkg.develop(PackageSpec(path=pwd()))
             Pkg.instantiate()'
+      - run: |
+          julia --project=docs -e '
+            using Pkg
+            using Documenter: doctest
+            using DiffImages
+            doctest(DiffImages)'
```

Comment:
Sorry that I didn't make it clear. I was thinking of adding `doctest(DiffImages)` into `test/runtests.jl`. See [OffsetArrays](https://github.com/JuliaArrays/OffsetArrays.jl/blob/7a43b4ca2b10a8c24a162f1365976d6a227a1666/test/runtests.jl#L42-L48) as an example.